test/distrib/bazel/run_bazel_distrib_test.sh

```diff
@@ -24,6 +24,12 @@ FAILED_VERSIONS=""
 for VERSION in $VERSIONS; do
     echo "Running bazel distribtest with bazel version ${VERSION}"
     ./test_single_bazel_version.sh "${VERSION}" || FAILED_VERSIONS="${FAILED_VERSIONS}${VERSION} "
+
+  if [ -n "$KOKORO_ARTIFACTS_DIR" ]; then
```

Comment:
this seems fragile to me. e.g. bazel can be configured to use different cache location and this trick will stop working.

How about making sure that we pass some option with different value for each bazel version under test, e.g. `--action_env=bazel_cache_invalidate=version_${VERSION}`? That should be sufficient for invalidating the cache between builds with different bazel version.

Reply:
Done.